Dockerfile

```diff
@@ -1,17 +1,18 @@
 FROM node
 
 ENV ROOT=/data
-ENV SRC_DIR=${ROOT}/src
+ENV SRC_DIR=${ROOT}
+
+WORKDIR ${SRC_DIR}
 
 COPY package.json package-lock.json ${SRC_DIR}/
 RUN npm install
-WORKDIR ${SRC_DIR}
-COPY . ${SRC_DIR}
+COPY src/ ${ROOT}/src
```

Comment:
If this container only does the backend src, why not just copy to `${ROOT}`?
If it will do more than just the backend why not just copy in `.` instead of `src/`

Reply:
We don't want te copy `./node_modules/` to `${ROOT}`, we don't need to copy the `Dockerfile` or the `docker-compose` or the `README` or anything else that is in the root of the project.

I think we are only interested in the `package.json` and the `package-lock.json`. But those are copied separately so docker can cache the `npm install` step. The other files we are interested in are all located in `./src/`.

We have to put it in `${ROOT}/src` or the command will no longer work. It now runs `npm run start`, but that script expects to find a `./src/back/main.js` file.